mne/io/constants.py

```diff
@@ -845,6 +848,11 @@ def __setattr__(self, attr, val):  # noqa: D105
 # KRISS sensors
 #
 FIFF.FIFFV_COIL_KRISS_GRAD         = 9001
```

Comment:
`FIFFV_COIL_SAMPLE_TMS_PLANAR` should be removed since it's removed in `coil_def.dat` now